### PR TITLE
Use rp_id as rp_name if rp_name is not defined

### LINKED ISSUE #40: Use RP ID as default RP name
Make `DJANGO_FIDO_RP_NAME` setting optional even for python-fido >= 0.15. Use RP ID, if RP name is not defined.

### PR DESCRIPTION
**Pull request by stinovlas**
Closes #40.

### CODE REVIEW

File: django_fido/views.py [hunk 1]
```diff
@@ -49,10 +49,11 @@ class Fido2ViewMixin(object):
     attestation = AttestationConveyancePreference.NONE
     session_key = FIDO2_REQUEST_SESSION_KEY
 
-    @property
-    def rp_name(self) -> Optional[str]:
+    rp_name = None  # type: Optional[str]
```
Comment (ziima) [inline django_fido/views.py]: Docstring wasn't updated.

File: django_fido/views.py [hunk 2]
```diff
@@ -49,10 +49,11 @@ class Fido2ViewMixin(object):
...
+
+    def get_rp_name(self) -> Optional[str]:
         """Return relying party name set in django settings."""
```
Comment (ziima) [inline django_fido/views.py]: `Return relying party name.`

File: django_fido/views.py (hunk 1 above)
Comment (stinovlas) [reply django_fido/views.py]: Updated.

File: django_fido/views.py (hunk 2 above)
Comment (stinovlas) [reply django_fido/views.py]: Updated.

**Pull request merged**